D/DuckDB/build_tarballs.jl

```diff
@@ -10,32 +10,48 @@ sources = [
     GitSource("https://github.com/duckdb/duckdb.git", "f680b7d08f56183391b581077d4baf589e1cc8bd"),
 ]
 
-# Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/duckdb/
 
-cmake -B build \
-      -DCMAKE_INSTALL_PREFIX=${prefix} \
-      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
-      -DCMAKE_BUILD_TYPE=Release \
-      -DENABLE_SANITIZER=FALSE \
-      -DBUILD_EXTENSIONS='autocomplete;icu;parquet;json;fts;tpcds;tpch' \
-      -DENABLE_EXTENSION_AUTOLOADING=1 \
-      -DENABLE_EXTENSION_AUTOINSTALL=1 \
-      -DBUILD_UNITTESTS=FALSE \
-      -DBUILD_SHELL=TRUE \
-      -DDUCKDB_EXPLICIT_PLATFORM="${target}"
-cmake --build build --parallel ${nproc}
-cmake --install build
+if [[ "${target}" == *-ming* ]]; then
+    # Special handling for windows build
+    [ -f CMakeCache.txt ] && rm CMakeCache.txt
+    cmake -G "Ninja" \
+        -DCMAKE_INSTALL_PREFIX=${prefix} \
+        -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DENABLE_SANITIZER=FALSE \
+        -DENABLE_EXTENSION_AUTOLOADING=1 \
+        -DENABLE_EXTENSION_AUTOINSTALL=1 \
+        -DBUILD_UNITTESTS=FALSE \
+        -DBUILD_SHELL=TRUE \
+        -DDUCKDB_EXPLICIT_PLATFORM="${target}" \
+        -DBUILD_EXTENSIONS="icu;parquet;json"
```

Comment:
You should keep `-B build` to keep build output in `build` (and that would match you `install` call).

I would recommend that you simplify to something like this (a single `cmake` call - with sorted args):
```suggestion
if [[ "$target" == *-w64-mingw32 ]]; then
  build_extensions="icu;parquet;json"
else
  build_extensions="autocomplete;icu;parquet;json;fts;tpcds;tpch"
fi

cmake -B build \
       -DBUILD_EXTENSIONS="$build_extensions" \
       -DBUILD_SHELL=TRUE \
       -DBUILD_UNITTESTS=FALSE \
       -DCMAKE_BUILD_TYPE=Release \
       -DCMAKE_INSTALL_PREFIX=$prefix \
       -DCMAKE_TOOLCHAIN_FILE=$CMAKE_TARGET_TOOLCHAIN \
       -DDUCKDB_EXPLICIT_PLATFORM="$target" \
       -DENABLE_EXTENSION_AUTOINSTALL=1 \
       -DENABLE_EXTENSION_AUTOLOADING=1 \
       -DENABLE_SANITIZER=FALSE
 cmake --build build --parallel $nproc
 cmake --install build

if [[ "$target" == *-w64-mingw32 ]]; then
  install -Dvm 755 build/src/libduckdb.$dlext -t $libdir
fi
```